Readme.md

```diff
@@ -3,7 +3,7 @@
 A interation of the Colony Dapp sporting both a fully decentralized operating mode, as well as a mode enhanced by a metadata caching layer
 
 ## Prerequisites
-* `node` `v14.18.x` (Best use [nvm](https://github.com/nvm-sh/nvm))
+* `node` `v16.14.x` (Best use [nvm](https://github.com/nvm-sh/nvm))
```

Comment:
Great, that you remembered about updating the readme

Reply:
Yeah I almost forgot to do it :P